Lib/test/test_frame.py

```diff
@@ -15,6 +15,7 @@
 from test import support
 from test.support import import_helper, threading_helper
 from test.support.script_helper import assert_python_ok
+from test import mapping_tests
```

Comment:
Maybe put this import with `from test import support`?

Reply:
There are three lines of `test.support` imports; I left them together.

Reply:
Ah I only meant to merge the imports at the same level. But it's up to you anyway.